packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -46,6 +48,10 @@ export async function processOutboundPatientDiscoveryResps({
 
     await processPatientDiscoveryProgress({ patient, status: "completed" });
 
+    const fullPatient = await getPatientOrFail({ id: patientId, cxId });
```

Comment:
What do you think we move this up and replace `patient` w/ it, so we don't need something called "fullPatient"? 😄

Reply:
To avoid having to query the db if there are no results i renamed the top instance to patientIds and this to patient